docs/pydoc/config/answer-generator.yml

```diff
@@ -12,15 +12,15 @@ processors:
   - type: smart
   - type: crossref
 renderer:
-   type: renderers.ReadmeRenderer
-   excerpt: Reads a set of documents and generates an answer to a question, word by word
-   category: 6310ca73c622850ddd3875a2
-   title: Answer Generator API
-   slug: answer-generator-api
-   order: 0
-   markdown:
-     descriptive_class_title: false
-     descriptive_module_title: true
-     add_method_class_prefix: true
-     add_member_class_prefix: false
-     filename: answer_generator_api.md
+  type: renderers.ReadmeRenderer
+  excerpt: Reads a set of documents and generates an answer to a question, word by word
+  category: 63f374c0c2ecdc004a43cd0a
```

Comment:
the category ID I get from [readme API ](https://docs.readme.com/main/reference/getcategories) is 63cffd5033018f0010bde74a...

Reply:
True that, I made a mistake. I didn't notice it didn't change in `main`.

I'll close this PR and only update `v1.14.x` branch.